apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -100,6 +100,7 @@ static int delete_task(int argc, char *argv[])
 */
 static int restart_task(int argc, char *argv[])
 {
+	sleep(SEC_1); /* task will be restarted during this time, so counter will not be incremented */
```

Comment:
same as previous. I don't know why this is necessary.

Reply:
Reverted.